modules/installing-rhacs-ocp-steps.adoc

```diff
@@ -10,25 +10,23 @@
 
 . On the {osp} cluster, install the {product-title-short} Operator into the `rhacs-operator` project, or namespace.
 . On the {osp} cluster that will contain Central, called the central cluster, use the {product-title-short} Operator to install Central services into the `stackrox` project. One central cluster can secure multiple clusters.
-. Log in to the {product-title-short} web console from the central cluster, and then create an init bundle and download it. The init bundle is then installed on the cluster that you want to secure, called the secured cluster.
+. Log in to the {product-title-short} web console from the central cluster, and then create an init bundle or cluster registration secret (CRS) and download it. The init bundle is then installed on the cluster that you want to secure, called the secured cluster.
 . For the secured cluster:
 .. Install the {product-title-short} Operator into the `rhacs-operator` namespace.
-.. On the secured cluster, apply the init bundle that you created in {product-title-short} by performing one of these steps:
-* Use the {ocp} web console to import the YAML file of the init bundle that you created. Make sure you are in the `stackrox` namespace.
-* In the terminal window, run the `oc create -f <init_bundle>.yaml -n <stackrox>` command, specifying the path to the downloaded YAML file of the init bundle.
+.. On the secured cluster, apply the init bundle or CRS that you created in {product-title-short} by performing one of these steps:
+* Use the {ocp} web console to import the YAML file of the init bundle or CRS that you created. Make sure you are in the `stackrox` namespace.
+* Do one of the following actions:
```

Comment:
How about the following to immediately show the difference between the two options:
```suggestion
* Use the {ocp} web console to import the YAML file of the init bundle or CRS that you created. Make sure you are in the `stackrox` namespace.
* Use the OpenShift CLI to perform one of the following actions:
```